.github/workflows/ci.yml

```
@@ -20,16 +20,15 @@ jobs:
         with:
           node-version: 14
 
-      - name: "Check that the cache files are consistent with their remote sources"
-        run: |
-          git diff --exit-code --quiet HEAD^ -- .yarn/cache || \
-            corepack yarn --immutable --immutable-cache --check-cache
-        if: |
-          github.event.pull_request != ''
+      - run: corepack yarn install
```

Non-blocking suggestion: IIUC, those flags are implicitly enable when Yarn 4 is run from a CI env, correct? While I agree it's a nice default, Iimo it's still worth explicitly adding the files so a human can run the command by copy pasting without needing to learn Yarn's defaults.
```suggestion
      - run: corepack yarn install --immutable --refresh-lockfile
```

---

Makes sense. We could add a comment explaining that.